edgelet/contrib/systemd/debian/iotedge.service

```diff
@@ -11,6 +11,7 @@ KillMode=process
 TimeoutStartSec=600
 TimeoutStopSec=40
 Restart=on-failure
+RestartPreventExitStatus=153
```

Comment:
Should the exit codes @shantanu1singh added also be listed here?

Reply:
I'm not sure. Looking at their names, it seems like they could be. @shantanu1singh, would it make sense to prevent restart for the three errors you mapped to status codes (`InvalidDeviceConfig`, `InvalidHubConfig`, `InvalidSignedToken`)? Could a restart potentially fix any of these problems, or do they all require some sort of intervention (new deployment, change to config.yaml, etc.)?

Reply:
Let's commit what we have, and we can add these to the list in a future commit if needed.